### ISSUE TITLE
[Uno] `RangeSelector` crashes Uno 5.1.87 apps with `The member BrushTransition.BrushTransition() is not implemented`

### ISSUE THREAD

**Issue by xperiandri** [CommunityToolkit/Windows#372]
### Describe the bug

Trying to render `RangeSelector` crashes Uno app UI with 3 exceptions related to `The member BrushTransition.BrushTransition() is not implemented`

### Steps to reproduce

```text
1. Create MVUX Uno app with login from template
2. Install `<PackageVersion Include="CommunityToolkit.WinUI.Controls.RangeSelector" Version="8.0.240109" />`
3. Add to Login page

      xmlns:wct="using:CommunityToolkit.WinUI.Controls"

      <wct:RangeSelector />
```

### Expected behavior

`RangeSelector` renders

### Screenshots

![image](https://github.com/CommunityToolkit/Windows/assets/2365592/6ac39470-1612-4944-b3f7-3bd45e0db257)

### Code Platform

- [ ] UWP
- [ ] WinAppSDK / WinUI 3
- [X] Web Assembly (WASM)
- [X] Android
- [X] iOS
- [X] MacOS
- [X] Linux / GTK

### Windows Build Number

- [ ] Windows 10 1809 (Build 17763)
- [ ] Windows 10 1903 (Build 18362)
- [ ] Windows 10 1909 (Build 18363)
- [ ] Windows 10 2004 (Build 19041)
- [ ] Windows 10 20H2 (Build 19042)
- [ ] Windows 10 21H1 (Build 19043)
- [ ] Windows 10 21H2 (Build 19044)
- [ ] Windows 10 22H2 (Build 19045)
- [ ] Windows 11 21H2 (Build 22000)
- [ ] Other (specify)

### App minimum and target SDK version

- [ ] Windows 10, version 1809 (Build 17763)
- [ ] Windows 10, version 1903 (Build 18362)
- [ ] Windows 10, version 1909 (Build 18363)
- [ ] Windows 10, version 2004 (Build 19041)
- [ ] Windows 10, version 2104 (Build 20348)
- [ ] Windows 11, version 22H2 (Build 22000)
- [ ] Other (specify)

### Visual Studio Version

2022

### Visual Studio Build Number

17.9.4

### Help us help you

Yes, but only if others can assist.

**Comment by Arlodotexe** [comment]
@xperiandri Could you try the nightly packages in our [MainLatest preview feed](https://github.com/CommunityToolkit/Windows/wiki/Preview-Packages#latest-)? Support for Uno 5.1 will arrive in the 8.1 release of the toolkit, I'd like to rule out version incompatibility errors first.

**Comment by xperiandri** [comment]
Indeed fixed in WCT 8.1
Tested with versoin `8.1.240322-build.993`
Thank you @Arlodotexe!

**Comment by xperiandri** [comment]
Is there any aproximate timeline for 8.1 release?
